portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,92 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Discover China</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Baloo+Tamma+2:wght@700&family=Lato:wght@300&display=swap" rel="stylesheet">
   </head>
+  
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Discover China 🇨🇳</h1>
+      <h2>你好! (Nǐ hǎo!)</h2>
+      <p><b>China</b> (中国), officially the People's Republic of China, is a country in East Asia. </p>
+      <!-- Information table -->
+      <table class="table">
+          <tbody>
+              <tr>
+                  <td><b>Capital</b></td>
+                  <td><a href="https://en.wikipedia.org/wiki/Beijing" target="_blank">Beijing (北京市)</a></td>
+              </tr>
+              <tr>
+                  <td><b>Area</b></td>
+                  <td>9,596,961 km<sup>2</sup></td>
+                </tbody>
+              </tr>
+              <tr>
+                  <td><b>Population</b></td>
+                  <td>1,400,050,000</td>
+              </tr>
+              <tr>
+                  <td><b>Currency</b></td>
+                  <td>Renminbi (yuan; ¥; CNY)</td>
+              </tr>
+              <tr>
+                  <td><b>Time zone</b></td>
+                  <td>UTC+8 (China Standard Time)</td>
+              </tr>
+
+          </tbody>
+        </table>
+      <br>  
+      
+      <!-- China's history youtube video -->
+      <iframe src="https://www.youtube.com/embed/YP1qjTzxQNE" title="The Animated History of China video"></iframe>  
+      
+      <h3>Some cool facts about China:</h3>
+      <ol>
+        <li>The Chinese New Year celebration lasts for 15 days.</li>
+        <li>China’s railway lines are so extensive and long that they could loop around Earth twice.</li>
+        <li>2,200 years ago, the game that later turned into football (soccer) was invented in China.</li>      
+      </ol>
+      <p>For further information please visit <a href="https://en.wikipedia.org/wiki/China" target="_blank">China's wikipedia page</a></p>
+      <h3>Now that we've covered the basics , let's look at some pictures from China:</h3>
+    </div>
+    
+    <!-- China's picture slideshow -->
+    <div class="slideshow-container">
+        <div class="slide fade">
+            <img src="Images/China-1.jpg" style="width:100%">
```

Comment:
Move all inline style declarations to the CSS file 
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns
```suggestion
            <img src="Images/China-1.jpg">
```

Reply:
done :)